File: AUGUST-2025/14-08-2025/p2264.dart
class Solution {
  String largestGoodInteger(String num) {
    List<String> value = ['999','888','777','666','555','444','333','222','111','000'];
    for(int i = 0 ; i < value.length ; i++){
        if(num.contains(value[i])){
            return value[i];
        }
    }
    return '';
  }
}

File: AUGUST-2025/14-08-2025/p2264.cs
public class Solution {
    public string[] good = new string[] {"999", "888", "777", "666", "555", "444", "333", "222", "111", "000"};

    public string LargestGoodInteger(string num) {
        foreach (string s in good)
            if (num.Contains(s))
                return s;
        return "";
    }
}